code/game/objects/machinery/mortar.dm

```diff
@@ -313,6 +324,13 @@
 	ai_targeter.forceMove(src.loc)
 	ai_targeter = null
 
+/obj/machinery/deployable/mortar/proc/handle_undeploy_references()
+	linked_mortar = get_internal_item()
+	LAZYINITLIST(linked_mortar.linked_binoculars)
+	LAZYCLEARLIST(linked_mortar.linked_binoculars)
+	for(var/binoc in src.linked_binoculars)
+		LAZYADD(linked_mortar.linked_binoculars, binoc)
```

Comment:
why not just set the vars to each other or use .Copy()?

Reply:
using .Copy() now